README.md

```diff
@@ -12,12 +12,59 @@ The only IRC Client you'll ever need, guaranteed!
 ## Missing Features
 
 Check the [issues](http://github.com/Chewsterchew/DIRCord/issues) to see what I'm planning next, or to suggest new things!
+## Requirements
 
-## Setup guide.
+* [Ruby (>= 2.4)](https://www.ruby-lang.org/en/)
+* [RVM](https://rvm.io/)
+* [Git](https://git-scm.com/)
 
-1) Create a server on Discord.
+## Setup Guide
 
-2) more soon lol
+*You no longer need to make a discord server, as the bot automatically makes one, however, it is left here for manual setup.*
```

Comment:
Remove this, split the Setup Guide into 2 parts, one where it uses setup.rb, the other, as is below

Reply:
okay